changelog.md

```diff
@@ -22,6 +22,7 @@ Saves from 6.0.0 are compatible with 6.1.0
 ## Fixes
 
 * **[Flight Planning]** Fixes CAS flights not having landing waypoints.
+* **[Mission Generation]** DCS built-in capture events are now recognized and processed in liberation again.
```

Comment:
Users won't know why this matters.

> Airbase capture is no longer blocked by grounded aircraft.

Reply:
> SGTM for a quick fix of the regression, but is this not also a bug for FOB captures? Won't a FARP with grounded helicopters need to have the helicopters destroyed before it can be captured?

Yes this issue still exists for FOBs. There is currently no issue in the Issue tracker for that but i will create one.
As a long term goal it would be better to have one reliable solution. This change is just about to bring in back the original functionality until there is a better solution :)

Reply:
> Users won't know why this matters.

You're right but there are some issue reports in discord so thats why i added it to the changelog. We can get rid of the changelog entry any time

Reply:
Yes, that's why I suggested the alternate phrasing :)

Reply:
ah 🤦 not recognized the change - thought it is a cite

Reply:
Sorry, I should have just used gitbub's change suggestion and it would have been a lot more clear :)